Add docs for My Builds

harrietgrace opened the pull request:
- [x]  Swap out placeholder image with real screenshot x 3
- [x]  Make the dot points into sentences
- [x]  Update the old screenshots on the Pipelines page so that they all include the my builds nav item (main gif, build history, speed, reliability, builds per week, and starring pipelines)

keithpitt commented on pages/pipelines/pipelines.md.erb:
```diff
@@ -62,4 +62,35 @@ Your pipeline’s name, description, its repository, and your default branch are
+
+### Past builds
+
+The My Builds page lists all of your past builds in reverse chronological order, as well as any that are currently running or waiting for agents. You can filter these to see only 'Running' or 'Scheduled' builds using the buttons in the top right corner. As you scroll down on this page, more builds from your Buildkite history will be loaded until they are all visible.   
```
>  As you scroll down on this page, more builds from your Buildkite history will be loaded until they are all visible

We don't do "infinite loading scrolling thingy" so there's a "Next Page" button I think at the bottom of the page. Probably don't need to mention this though!

harrietgrace replied:
👌🏻

keithpitt commented:
We probably also need to add a seciton on "Linking builds to your account" or "Why aren't my builds showing" that talks about how we use the Git email address `git config --get user.email` and users's email to link them all together?

Maybe it can be it's now "My Builds" docs page as well? That'd give us a consistent thing to link in the UI and in support if we've got any questions about builds not showing up.

harrietgrace commented on pages/pipelines/pipelines.md.erb:
```diff
@@ -62,4 +62,35 @@ Your pipeline’s name, description, its repository, and your default branch are
 
+## My Builds
+
+In contrast (something softer, like conjunction) with the high level pipeline information on the Pipelines page, the My Builds dropdown menu provides a personalised list of your most recently triggered builds.   
```
`Along with` is perfect, many thanks 👏🏻

harrietgrace commented on pages/pipelines/pipelines.md.erb:
```diff
@@ -62,4 +62,35 @@ Your pipeline’s name, description, its repository, and your default branch are
+
+### Navigation item
+
+Living in the navigation bar, the My Builds menu is directly accessible from every page in the Buildkite web application. The count that appears in the My Builds navigation bar item tells you how many of your builds are running or waiting to run.
```
Uhh, not sure. I think probably just Buildkite... I was thinking about how sometimes we need to make the distinction between stuff you can access with the buildkite-agent command and the stuff you can get to in the UI. But that isn't this section, so I think just 'Buildkite' would be good here.

harrietgrace commented:
Ah cool, I didn't even think about that bit. I was hoping to keep this info close to the Pipelines overview, because it's kind of part of that. But also very much Builds related... so it could either sit under the Pipelines heading, or it can go down into Builds. I'd like to keep it in Pipelines, but then I would feel weird about it having a title of My Builds in the sidebar.

keithpitt commented:
Why not both!?

You can have a dedicated "My Builds" page with a link to it from the pipelines page?

toolmantim commented on pages/builds/troubleshooting-builds.md.erb:
```diff
@@ -0,0 +1,11 @@
+
+## The build that was created from my Pull Request failed immediately after starting
+
+Check to make sure that the email address attached to your git user account has permission to create builds in your Buildkite organisation. When a build is created by a webhook from somewhere like Github, the first thing Buildkite will do is check the permissions of the author of the most recent commit. If that author's email isn't linked to the Buildkite organisation you're creating a build unde ...
```
Ah, this isn't actually a thing I don't think! Unless I'm mistaken. What did you see that led you to believe this was how it worked? I might have missed something.

harrietgrace replied:
This happened when I pushed my first build when I joined BK, and it was because I hadn't added my git email address...

toolmantim replied:
Ah, I think this has changed since then. It might have been related to teams. Or perhaps to build triggering… but I think we're planning on changing that too! So for now I'd just assume that builds won't failed because of unmatched emails.

toolmantim commented on pages/quickstart/getting_started.md.erb:
```diff
@@ -16,6 +16,16 @@ We’ve created a simple [bash-example test repository](https://github.com/build
 
+## Link your git email address to Buildkite
+
+Before creating a build, link the email address you have associated with your git user account to Buildkite. Until you've linked your git email address, any builds created on commits you've authored won't have the correct permissions and will fail. 
```
This isn't true, they'll still work! And they'll be able to just hit the "Add This Email" button with the new tech that Jess has been working on… they don't need to do it ahead of time.

harrietgrace replied:
Okay cool, just remove this section then?

harrietgrace replied:
Orrr update it to say how to use the little popup?

toolmantim replied:
Yeah we could update it with a little "If Buildkite doesn't recognise your Git email, you'll get this thing which will fire unicorn dust and make everything better again"?

harrietgrace commented:
@toolmantim Could you please take another squiz at this? I moved the troubleshooting section back into the My Builds page, and removed the question that wasn't relevant. Also updated the Getting Started doc's section about linking emails to mention the My Builds menu and removed the bit about builds failing without the correct email.

harrietgrace commented on pages/builds/my_builds.md.erb:
```diff
@@ -0,0 +1,36 @@
+
+### The build I just created is running, but isn't showing up in the My Builds menu
+
+The most common reason for a build to not be included in your My Builds menu is that the email address isn't linked to your account. Make sure that the email attached to the git user that you used to trigger the build is linked to your Buildkite account. Without this link, we won't know that the most recent build is yours!
```
Something different for the bit about git emails

harrietgrace commented:
@toolmantim new version of the troubleshooting section for you. Also those styles are 💥
